extension-style/src/main/java/com/mapbox/maps/extension/style/expressions/dsl/generated/ExpressionDsl.kt

```diff
@@ -192,7 +192,7 @@ fun floor(block: Expression.ExpressionBuilder.() -> Unit) = Expression.floor(blo
 fun format(block: Expression.FormatBuilder.() -> Unit) = Expression.format(block)
 
 /**
- * Returns the feature's geometry type: `Point`, `MultiPoint`, `LineString`, `MultiLineString`, `Polygon`, `MultiPolygon`. `Multi*` feature types are only returned in GeoJSON sources. When working with vector tile sources, use the singular forms.
+ * Returns the feature's geometry type: `Point`, `LineString` or `Polygon`. `Multi*` feature types return the singular forms.
```

Comment:
wondering where this change comes from. we have only updated `promoteId` docs in stylegen pr. isn't it?

Reply:
This happened due to previous changes that we forgot to apply

Reply:
Turns out it was my previous update, must be that I've only generated new docs in the v10.9 branch